rosdep/python.yaml

```diff
@@ -8768,6 +8768,14 @@ python3-pytest-mock:
       packages: [pytest-mock]
   rhel: ['python%{python3_pkgversion}-pytest-mock']
   ubuntu: [python3-pytest-mock]
+python3-pytest-order:
+  arch: [python-pytest-order]
+  debian: [python3-pytest-order]
+  gentoo: [dev-python/pytest-order]
+  nixos: [python3Packages.pytest-order]
```

Comment:
Not sure what to put here, as nix namespaces the package with `311` or `312`, not just `3`.

Reply:
I think they have "aliases", or something like that, that make this work.  CI seems happy enough with it.

That said, nixos isn't a required platform, so if you are uncomfortable with it, just remove it.

Reply:
If CI is happy, I'm fine.